pandas/tests/groupby/test_transform.py

```diff
@@ -743,14 +754,21 @@ def get_result(grp_obj):
                                   limit=limit)
 
     if test_series:
-        exp = pd.Series(exp_vals * 2)
-        exp.name = 'vals'
+        exp = exp.loc[:, 'A']
         grp = grp['vals']
         result = get_result(grp)
+        df.insert(0, 'A', result)
```

Comment:
What is the point of all of the insert / sort calls? Would rather just construct the expected value and simply compare that to the result of a pct_change call

Reply:
Agree. Fixed.